src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,26 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo/CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+type Props = {
+  comments: Comment[] | null;
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => {
+  if (comments) {
```

Comment:
check it on the higher level, there is no sense to render CommentList component if we don't have comments